src/components/maps/LineMap.vue

```diff
@@ -153,7 +153,7 @@
     const lineSeries = chart.series.push(am5map.MapLineSeries.new(root, {}))
 
     lineSeries.mapLines.template.setAll({
-      stroke: am5.color(getColor(themeColors.value?.primary)),
+      stroke: am5.color(getColor(themeColors.value.variables?.primary)),
```

Comment:
getColor('primary')

or themeColors.value.variables.primary

One of this way to get color, do not combine them.